config.go

```diff
@@ -184,8 +191,121 @@ func (c *Config) getDefaults() {
 	}
 }
 
-// CreatePythonVenv creates a virtualenv for Python scripts in ~/.ralph-cli dir.
-func CreatePythonVenv() error {
-	// TODO(xor-xor): Implement this.
+// Manifest represents the contents of a .toml file holding additional information,  which may be
+// helpful/required to run user's script (e.g., language, version, requirements etc.).
+type Manifest struct {
+	Path            string `toml:"-"`
+	Language        string
+	LanguageVersion int
+	Requirements    []requirement `toml:"requirement"`
+}
+
+// requirement is a helper type for Manifest. It shouldn't be used alone/separately.
+type requirement struct {
+	Name    string
+	Version string
+}
+
+// GetManifest loads manifest file pointed by path argument. Each manifest file should have the same
+// name as the script file associated with it (e.g., having a script "idrac.py", its manifest file
+// should be named "idrac.toml").
+// At this moment manifest files are not required, although it may change in the future (especially
+// when we add some more information to them).
+func GetManifest(path string) (*Manifest, error) {
+	var mf Manifest
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return nil, nil
+	}
+	if _, err := toml.DecodeFile(path, &mf); err != nil {
+		return nil, fmt.Errorf("error reading manifest file %s: %v", path, err)
+	}
+	mf.Path = path
+	mf.Language = strings.ToLower(mf.Language)
+	if err := mf.validate(); err != nil {
+		return nil, fmt.Errorf("%v", err)
+	}
+	return &mf, nil
+}
+
+// validate performs some sanity checks and normalizations on Manifest.
+func (m *Manifest) validate() error {
```

Comment:
I'm not sure if error is proper type for handling validation. I understand that this error are meant for user who provides invalid config. I'd like to get all errors at once rather then fixing them one by one. Not huge issue but important from UX perspective.